composer.json

```diff
@@ -38,7 +38,7 @@
         "cweagans/composer-patches": "^1.6",
         "drupal-composer/drupal-scaffold": "^2.2",
         "drupal/console": "^1.0.1",
-        "drupal/core": "~8.0",
+        "drupal/core": "~8.3",
         "drush/drush": "~8.0",
         "php": ">=5.5.9",
```

Comment:
You should remove the php requirement.. Drupal's PHP requirement is already in Drupal's composer.json.